tests/core/audit_test.py

```diff
@@ -475,6 +475,228 @@ def new_baseline(self):
         }
 
 
+class TestDetermineAuditResults(object):
+
+    @pytest.fixture
+    def mock_get_raw_secret_value(self):
+        with mock.patch.object(
+            audit,
+            'get_raw_secret_value',
+            autospec=True,
+        ) as _mock:
+            yield _mock
+
+    @pytest.fixture
+    def mock_get_git_sha(self):
+        with mock.patch(
+            'detect_secrets.core.audit.get_git_sha',
+            return_value=None,
+            autospec=True,
+        ) as _mock:
+            yield _mock
+
+    @pytest.fixture
+    def mock_get_git_remotes(self):
+        with mock.patch(
+            'detect_secrets.core.audit.get_git_remotes',
+            return_value=None,
+            autospec=True,
+        ) as _mock:
+            yield _mock
+
+    def get_audited_baseline(self, plugin_config={}, is_secret=None):
```

Comment:
Since `plugin_config` is always passed, afaict, I don't think you need the default value. (There's also the python gotcha w/ dict's as a default arg.)

Also you can do e.g. `get_audited_baseline(plugin_config=foo, is_secret=True)` rather than `get_audited_baseline(foo,True)` as it's a little more readable what the args are from seeing the call.

Reply:
Makes sense, fixed